xmrig/common/xmrig.h

```diff
@@ -61,20 +61,22 @@ enum AlgoVariant {
 
 
 enum Variant {
-    VARIANT_AUTO = -1, // Autodetect
```

Comment:
seems like extra spaces added for new lines. Maybe remove to have minimal set of the changes?

Reply:
Yeah, I thought about it, but I tried to keep alignment since `VARIANT_WALTZ` is longer than other on one symbol